play.pokemonshowdown.com/src/client-main.ts

```diff
@@ -55,12 +56,10 @@ class PSPrefs extends PSStreamModel<string | null> {
 	 * null - Enable GIFs only on Chrome 64.
 	 */
 	nogif: boolean | null = null;
-
```

Comment:
You deleted three lines here. By accident? Either way they shouldn't be deleted.